lakefile.lean

```diff
@@ -2,30 +2,22 @@ import Lake
 
 open Lake DSL
 
-def moreServerArgs := #[
-  "-Dpp.unicode.fun=true", -- pretty-prints `fun a ↦ b`
```

Comment:
Should we keep this comment, or is it boring? Does hovering over `` `pp.unicode.fun`` display the docstring for the option?

Reply:
Can we worry about that later, or separately?

Reply:
Hovering says "A literal of type `Name`."

Reply:
Let's keep the comment for now then; clearly something thought it was useful, and we can decide to remove it later separately.

Reply:
(but pinging @tydeu about the feature request of option docstrings in vscode hovers here)